### DISCUSSION TITLE
Is Codespaces error "Unable to forward localhost:8080. Network Error"

### DISCUSSION THREAD

**Discussion by alexec** [community/community#68550] [Codespaces]
### Select Topic Area

Bug

### Body

Port forwarding seems intermittently broken. Yesterday it was working, but today it is not. 

I tried two different Java Spring Boot repositories. I'm not sure if it is my account or a more general issue?

<img width="554" alt="image" src="https://github.com/community/community/assets/1142830/87ba6289-43e8-45d7-be31-20d03adc3e12">

**Comment by Primus66** [comment]


**Reply by alexec** [reply]
I don't see how that could be anything to do with Codespaces. Can you be more specific please?

**Comment by AnimMouse** [comment]
Unblock this domain in your firewall: `tunnels.api.visualstudio.com`

**Comment by alexec** [comment]
This discussion duplicates [this dicuss](https://github.com/orgs/community/discussions/68324#discussioncomment-7208534)
